langchain/vectorstores/opensearch_vector_search.py

```diff
@@ -250,13 +264,26 @@ def add_texts(
 
         Returns:
             List of ids from adding the texts into the vectorstore.
+
+        Optional Args:
+            vector_field: Document field embeddings are stored in. Defaults to
+            "vector_field".
+
+            text_field: Document field the text of the document is stored in. Defaults
+            to "text".
         """
-        embeddings = [
-            self.embedding_function.embed_documents([text])[0] for text in texts
-        ]
+        embeddings = self.embedding_function.embed_documents(list(texts))
         _validate_embeddings_and_bulk_size(len(embeddings), bulk_size)
+        vector_field = _get_kwargs_value(kwargs, "vector_field", "vector_field")
```

Comment:
This is a meta-comment on a previous PR, but `_get_kwargs_value` could just be `kwargs.get(key, default_value)`

Reply:
@vowelparrot there is a PR addressing this https://github.com/langchain-ai/langchain/pull/13184